index.html

```diff
@@ -47,11 +49,10 @@ <h1 class="textheader">Do You Want to Do Something?</h1>
                         <option value="diy">DIY</option>
                         <option value="music">Music</option>
                     </select>
-                </label>
-            </div>
-            <div class="button">
-                <button class="searchButton">Search</button>
-            </div>
+                </div>
+                <div class="button">
```

Comment:
I think you can get rid of this wrapper and just style the button itself, and I would say take advantage of the color scheme that you have come up with for this site and the visual language of the other elements on the page.
What does the button look like when it isn't being used? What about when you hover over it? What about while you're clicking on it?